### PR TITLE
Added project with mapping abstractions.

### PR DESCRIPTION
**Pull request by marisks**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,16 +1,121 @@
...
+public class MyPocoToMyDtoMapper : IMapper<MyPoco, MyDto>
+{
+    public virtual void Map(MyPoco from, MyDto to)
+    {
+        to.Name = from.Name;
+    }
+}
+```
+
+This mapping implementation will work for any classes, even for those that has no parameter-less constructor.
```
Comment (valdisiljuconoks) [inline README.md]: ".. has constructor with parameters."

File: README.md [hunk 2]
```diff
@@ -1,16 +1,121 @@
...
+public class MyPocoToMyDtoMapper : Mapper<MyPoco, MyDto>
+{
+    public override void Map(MyPoco from, MyDto to)
+    {
+        to.Name = from.Name;
+    }
+}
+```
+
+Now you can use these mapper by injecting it.
```
Comment (valdisiljuconoks) [inline README.md]: "...these mappers" or "...this mapper"

File: README.md [hunk 3]
```diff
@@ -1,16 +1,121 @@
...
+
+When you want to map one object to other, then use `IMapper<TFrom, TTo>` interface.
```
Comment (valdisiljuconoks) [inline README.md]: "...one object to other (if have already instances of these classes)"

**Comment by valdisiljuconoks** [comment]
Also for at least newly added files you should add copyright / license file header.
Template here - https://github.com/Geta/development-guidelines/blob/master/package-guidelines.md#file-header

File: README.md (hunk 3 above)
Comment (marisks) [reply README.md]: I think additional comments not necessary here. The object is already an instance of the class.

**Pull request merged**